packages/ember-runtime/tests/legacy_1x/mixins/observable/observable_test.js

```diff
@@ -129,8 +129,8 @@ QUnit.test('should return null when property value is null on Ember.Observable',
 });
 
 QUnit.test('should call unknownProperty when value is undefined on Ember.Observable', function() {
-  equal(get(object, 'unknown'), 'unknown');
```

Comment:
This would fail if only the `Ember.get()` module was run and relied on the `object.get()` module being run first to pass—for obvious reasons.